apps/ledger-live-mobile/android/app/build.gradle

```diff
@@ -221,10 +221,6 @@ dependencies {
     // implementation 'com.brentvatne.react:react-native-video'
     implementation project(':react-native-video')
     implementation "androidx.appcompat:appcompat:1.0.0"
-
-    // Adjust
-    implementation 'com.google.android.gms:play-services-analytics:10.0.1'
-    implementation 'com.android.installreferrer:installreferrer:1.0'
```

Comment:
same question

Reply:
I checked for Storyly, Braze and Segment. It's good for those three